index.html

```diff
@@ -232,39 +235,393 @@ <h3>Create</h3>
+            </li>
+            <li>
+Ensure the proper key length of <em>raw public key bytes</em> based on the
+<em>multicodec value</em> table provided below:
+              <table class="simple">
+                <thead>
+                  <th>Multicodec hexadecimal value</th>
+                  <th>public key byte length</th>
+                  <th>Description</th>
+                </thead>
+
+                <tbody>
+                  <tr>
+                    <td>`0xe7`</td>
+                    <td>33 bytes</td>
+                    <td>secp256k1-pub - Secp256k1 public key (compressed)</td>
+                  </tr>
+                  <tr>
+                    <td>`0xec`</td>
+                    <td>32 bytes</td>
+                    <td>x25519-pub - Curve25519 public key</td>
+                  </tr>
+                  <tr>
+                    <td>`0xed`</td>
+                    <td>32 bytes</td>
+                    <td>ed25519-pub - Ed25519 public key</td>
+                  </tr>
+                  <tr>
+                    <td>`0x1200`</td>
+                    <td>33 bytes</td>
+                    <td>p256-pub - P-256 public key (compressed)</td>
+                  </tr>
+                  <tr>
+                    <td>`0x1201`</td>
+                    <td>?? bytes</td>
+                    <td>p384-pub - P-384 public key (compressed)</td>
+                  </tr>
+                  <tr>
+                    <td>`0x1202`</td>
+                    <td>?? bytes</td>
+                    <td>p521-pub - P-521 public key (compressed)</td>
+                  </tr>
+                  <tr>
+                    <td>`0x1205`</td>
+                    <td>?? bytes</td>
+                    <td>
+rsa-pub - RSA public key. DER-encoded ASN.1 type RSAPublicKey according to
+IETF RFC 8017 (PKCS #1)
+                    </td>
+                  </tr>
+                </tbody>
+              </table>
+If the byte length of <em>raw public key bytes</em> does not match the expected
+public key length for the associated <em>multicodec value</em>, an
+INVALID_PUBLIC_KEY_LENGTH error MUST be raised.
+            </li>
+            <li>
+Set the <em>verification method</em> `id` value by concatenating
+<em>identifier</em>, a hash character (`#`), and the <em>multicodec value</em>.
+            </li>
+            <li>
+Set the <em>key encoding type</em> value to the <em>options</em> `type` value
+if it is set, or `Multikey` if it is not set.
+            </li>
+            <li>
+If <em>options</em> `allowExperimentalPublicKeyTypes` is not set to `true` and
+<em>key encoding type</em> is not `Multikey`, `JsonWebKey2020`, or
+`Ed25519VerificationKey2020`, an INVALID_PUBLIC_KEY_TYPE error MUST be raised.
+            </li>
+            <li>
+Set the <em>verification method</em> `type` to the <em>key encoding type</em>.
+            </li>
+            <li>
+Set the <em>verification method</em> `controller` to <em>identifier</em>.
```

Comment:
I suggest being more explicit about DID vs DID URL here... seen a lot of first timer errors related to this.

Reply:
Hrm, good point. What are the first timer errors you're seeing so I can try to craft some language in that direction? 

It feels like this sort of language is way more important for did:web than it is for did:key... but I'm probably missing some nuance in the point you're making.

Reply:
Got it, I'll put in language to make those MUSTs explicit.

Reply:
Fixed in a6aa2cd643afd1e46a865f648ceee0de9ef973e9.